packages/deployment-server-ui/src/pages/Devices.tsx

```diff
@@ -1,9 +1,90 @@
+import { Device, DeviceType } from ".prisma/client";
+import { Table } from "antd";
+import { useEffect, useState } from "react";
 import defineFunctionalComponent from "../util/defineFunctionalComponent";
 
 export default defineFunctionalComponent(function Devices() {
+  const [devices, setDevices] = useState<Device[]>(
+    Array(15).fill({})
+  );
+  const [deviceTypes, setDeviceTypes] = useState<DeviceType[]>(
+    Array(15).fill({})
+  );
+
+  const [filters, setFilters] = useState<any>({});
```

Comment:
Types are missing, please define

Reply:
![image](https://user-images.githubusercontent.com/26458909/149431092-cbfdca19-5697-4378-981b-70eb182fcba5.png)

can you help me with the correct Type syntax of these I understand it as:
{Int, Int} 
{String[], String[]}

Reply:
I can't tell for sure, but the only think that makes sense given the context is something like:

```suggestion
  const [filters, setFilters] = useState<{status: string[], deviceTypeId: string[]}>({status: [], deviceTypeId: []});
```

it may also be 

```ts
  const [filters, setFilters] = useState<{status: string[] | null, deviceTypeId: string[] | null}>({status: null, deviceTypeId: null});
```

or something similar, depending on how antd wants to represent the state where no filters are applied